chart/values.yaml

```diff
@@ -92,6 +94,13 @@ db:
   jwtSecretKeyRef:
     name: incident-commander-postgrest-jwt
     key: PGRST_JWT_SECRET
+  connectionPooler:
+    enabled: "{{ .Values.global.connectionPooler.enabled }}"
```

Comment:
this will not work, enabled should default to false in charts and then when installing the chart you would set global.db.connectionPooler.enabled=true